website/docs/getting-started/setup/upgrade-to-business-edition/docker.mdx

```diff
@@ -0,0 +1,129 @@
+---
+description: Follow the steps to upgrade the Appsmith community docker installation to a Business Edition.
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Docker
+
+Follow the steps listed below to **upgrade** your Community Edition (CE) to a Business Edition (EE) installed on docker.
+
+### Step 1: Database backup
+
+The Community and Business editions use Mongo DB, so you don't have to migrate the database. It's still recommended to [backup the Community Edition Mongo DB](/getting-started/setup/instance-management/appsmithctl#export-database).
+
+> Ensure that you backup the database with the docker instance **up** and **running**.
+
+### Step 2: Add or replace license key
+
+Go to the directory `~stacks/configuration` and open the `docker.env` file. Add/replace the Business Edition license key and save the file.
+
+```bash
+APPSMITH_LICENSE_KEY=<ENTER_THE_LICENSE_KEY>
+```
+
+### Step 3: Update image
+Go to the directory where the `docker-compose.yml` file is located. Open the `docker-compose.yml` file, and look for the `image:` key. Update the image in the `docker-compose.yml` file from Community Edition (_appsmith-ce_) to Business Edition (_appsmith-ee_). 
+
+The updated file should have the image attribute set as below:
+
+ ``` yaml
+  services:
+    appsmith:
+    #highlight-next-line
+      image: index.docker.io/appsmith/appsmith-ee
+```
+And save the file.
+
+> If you are using `docker run`, head to [recreate the instance](#step-4-recreate-the-instance) step.
```

Comment:
I think we should avoid quoting text for important info unless it is universal.  I think initially, we discussed this as a team and thought it would be a good idea. But somehow I feel it looks odd. Maybe we can emphasize the text that we want to bring attention to by making it bold? So the below should work

> If you are using the **Docker run** command, skip to [Step 4](https://appsmith-docs-git-rehaul-upgrade-business-edition-get-appsmith.vercel.app/getting-started/setup/upgrade-to-business-edition/docker/#step-4-recreate-the-instance).

You can bold Docker run above to emphasize

Also, this line should be placed at the beginning so that users can directly jump to step 4, if they are using Docker run instead of finding out after running the commands in step 3.